x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/CrossClusterQueryIT.java

```diff
@@ -1039,10 +1039,7 @@ public void testNoBothIncludeCcsMetadataAndIncludeExecutionMetadata() throws Exc
 
         assertThat(
             expectThrows(VerificationException.class, () -> runQuery(request)).getMessage(),
-            containsString(
-                "Both [include_execution_metadata] and [include_ccs_metadata] query parameters are set. "
-                    + "Use only [include_execution_metadata]"
-            )
+            containsString("Both [include_execution_metadata] and [include_ccs_metadata] query parameters are set. " + "Use only one")
```

Comment:
NIT: unnecessary string concatenation:
```suggestion
            containsString("Both [include_execution_metadata] and [include_ccs_metadata] query parameters are set. Use only one")
```